kbase-extension/static/kbase/templates/data_staging/ftp_file_table.html

```diff
@@ -13,54 +13,11 @@
         </tr>
     </thead>
     <tbody class="kb-staging-table-body">
-    {{#each files}}
```

Comment:
This all gets done directly by DataTables now. Might consider doing away with the whole template in the future.